src/NUnitFramework/tests/nunit.framework.tests-sl-5.0.csproj

```diff
@@ -297,7 +297,6 @@
     <Compile Include="Syntax\TestCompiler.cs" />
     <Compile Include="Syntax\ThrowsTests.cs" />
     <Compile Include="Syntax\TypeConstraints.cs" />
-    <Compile Include="TestContextTests.cs" />
```

Comment:
Slip of the mouse. :-) I found it missing in other projects and added the file back, but I didn't notice it was no longer in Silverlight.